internal/task/task/fs/mount.go

```diff
@@ -263,7 +263,7 @@ func NewMountFSTask(curvradm *cli.CurveAdm, cc *client.ClientConfig) (*task.Task
 		ContainerId:    &containerId,
 		ExecWithSudo:   false,
 		ExecInLocal:    true,
-		ExecTimeoutSec: 10,
+		ExecTimeoutSec: 3,
```

Comment:
Is this enough? how about add a timeout option with 3 seconds default for mount task? like:
```
curveadm mount NAME_OF_CURVEFS MOUNT_POINT --timeout 10
```